lib/src/telegram/model.g.dart

```diff
@@ -59,17 +59,17 @@ Map<String, dynamic> _$UpdateToJson(Update instance) {
   }
 
   writeNotNull('update_id', instance.update_id);
-  writeNotNull('message', instance.message);
```

Comment:
maybe run `./build.sh` again after you make the other changes, just to make sure 😃

Reply:
I ran it, don't know why that happened 😉